patches/tModLoader/Terraria/Terraria.csproj.patch

```diff
@@ -82,6 +82,12 @@
      <EmbeddedResource Include="Localization/Content/**" />
      <EmbeddedResource Include="Microsoft/**" />
    </ItemGroup>
++  <ItemGroup>
++    <None Remove="ModLoader\Default\UnloadedDresser.png" />
++  </ItemGroup>
+   <Target Name="EditBin" AfterTargets="Build">
```

Comment:
Uh.

Reply:
Fixed. Removed lines; not sure how that got in there, but they're gone

Reply:
Re: fixed. File no longer a part of PR